script/lint.js

```diff
@@ -170,8 +176,6 @@ const LINTERS = [{
         DEPOT_TOOLS_WIN_TOOLCHAIN: '0',
         ...process.env
       };
-      // Users may not have depot_tools in PATH.
-      env.PATH = `${env.PATH}${path.delimiter}${DEPOT_TOOLS}`;
```

Comment:
This is already handled (correctly) on L21. Environment variables on Windows are case-insensitive, and `process.env` handles that correctly. The issue here was that `process.env` has been destructured into a plain object above, so the special case-insensitive behavior was no longer applying, leading to `env.PATH` being undefined. Since L21 accesses it directly at `process.env.PATH`, it works as intended.